web/my-transcriptions.html

```diff
@@ -302,6 +302,16 @@ <h2 class="pane-title"> Contact </h2>
                     <a class="deleteBtn" onclick="deleteProject(${b.id}, event);" title="Delete Transcription">&cross;</a>
                 </div>`, ``)
         }
+        else{
```

Comment:
Also need to hide the "To continue working on a transcription, click the manuscript name. Remove by clicking the "✗" button." message